spec/models/stripe_wrapper_spec.rb

```diff
@@ -0,0 +1,64 @@
+require 'spec_helper'
+
+describe StripeWrapper do 
+  describe StripeWrapper::Charge do 
+    describe ".create" do 
+      it "makes a successful charge", :vcr do 
+        token = Stripe::Token.create(
+          :card => {
+            :number => "4242424242424242",
+            :exp_month => 6,
+            :exp_year => 2018,
+            :cvc => 314
+          }
+        ).id
+
+        response = StripeWrapper::Charge.create(
+          amount: 999,
+          source: token,
+          description: "a valid charge"
+        )
+
+        expect(response).to be_successful
+      end 
+
+      it "makes a card decline a charge", :vcr do 
```

Comment:
You could refactor these and put `:vcr` tag above, to `describe ".create"` level.